### DISCUSSION TITLE
Is IPV6  supported in comment system?

### DISCUSSION THREAD

**Discussion by Jimmi08** [e107inc/e107#5175] [Q&A Core] [answered]
Hi,
I am creating only a discussion, not an issue because I am not able to test it on the latest GitHub version. Maybe later. 

I tried to approve a comment on my site and I couldn't because of the IP address. 

![image](https://github.com/e107inc/e107/assets/5429548/a1e573a7-e298-4af4-a3d6-34382d992b82)

Maybe somebody other has this problem too.

**Comment by Deltik** [comment]
This is a bug, specifically on [this line](https://github.com/e107inc/e107/blob/d1f1b9e7cc213882a6027a0f50010f8a21f847f5/e107_handlers/form_handler.php#L6618-L6618):

With the `:` separators, the maximum length of an IPv6 address should be 39 characters, but the text box is incorrectly limited to 32 characters.

**Reply by Deltik** [reply, marked as answer]
Issue created: https://github.com/e107inc/e107/issues/5176
